design/pv_backup_info.md

```diff
@@ -42,21 +42,23 @@ The 1 version of `VolumeInfo` definition is:
 type VolumeInfoV1 struct {
     PVCName        string    // The PVC's name. The format should be <namespace-name>/<PVC-name>
     PVName         string    // The PV name.
-    BackupMethod   string    // The way the volume data is backed up. The valid value includes `VeleroNativeSnapshot`, `PodVolumeBackup`, `CSISnapshot` and `Skipped`.
-    SnapshotDataMovement bool // Whether the volume's snapshot data is moved to specified storage.
+    BackupMethod   string    // The way the volume data is backed up. The valid value includes `VeleroNativeSnapshot`, `PodVolumeBackup` and `CSISnapshot`.
```

Comment:
just wondering if in future we might support multiple forms of backup for 1 PV.

Reply:
There are some tries in that direction, but not in exactly backing a volume with multiple way's directions, for example, the snapshot data mover, which backs up the volume by snapshot, then backs up by the filesystem uploader.

IMO, maybe there will not be a universal way to let volume back up data by arbitrary ways that Velero can provide because this can be done by separate backups, but if there is a valid using scenario, we can try to implement it with proper ways.